### ISSUE TITLE
"A Calmer You" theme uses a wrong color, and themed icon is off-center

### ISSUE THREAD

**Issue by Xorio71** [Jays2Kings/tachiyomiJ2K#1701]
### Steps to reproduce

_For the "A Calmer You" theme bug:_

1. Enter TachiJ2K
2. Select the theme
3. Look at the bottom bar, and pay attention to its color. Then compare it to the color used in vanilla Tachi (I'll attach images for ya, don't worry), which is correct based on Material You guidelines.

_For the themed icon issue:_

Just look at it and compare it to the normal icon. The themed one should be smaller in size, to correspond to the size of the kanji in the normal icon, and at the same time, I feel like it should be a little higher as well, so it can be centered.
![Screenshot_20231226-151749~2](https://github.com/Jays2Kings/tachiyomiJ2K/assets/101265862/ac9b2c8e-2a3b-499b-8e64-50bc162d2f69)
![Screenshot_20231226-151641~2](https://github.com/Jays2Kings/tachiyomiJ2K/assets/101265862/d27b99f3-820f-4c93-837e-27d56d26bc24)
![Screenshot_20231226-152008~2](https://github.com/Jays2Kings/tachiyomiJ2K/assets/101265862/b86eff6b-56b5-479a-9a84-794e4cde9f90)

### Expected behavior

The bottom bar should have the colors that it has in the normal Tachiyomi app.

The kanji in the themed icon should be a bit smaller and moved up.

### Actual behavior

The bottom bar has an untinted gray color.

The kanji is too big and very close to the bottom half of the circle.

### TachiyomiJ2K version

1.7.3

### Android version

Android 14

### Device

Redmi Note 9 Pro

### Other details

Using an Android 14 custom ROM named PixelBuilds. No modifications to it from my part, and no modifications to monet from its.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I am reporting an issue exclusive to this fork. I have also checked that is not an issue on the [main version of Tachiyomi](https://github.com/tachiyomiorg/tachiyomi)
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[1.7.3](https://github.com/Jays2Kings/tachiyomiJ2K/releases/latest)**.
- [X] I have updated all installed extensions.
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] I have filled out all of the requested information in this form.

**Comment by CarlosEsco** [comment]
The bottom nav color is a design choice, it's not expected to match the main app 1 to 1

**Comment by Xorio71** [comment]
If it is a design choice, it doesn't look all that great.

**Comment by CarlosEsco** [comment]
> If it is a design choice, it doesn't look all that great.

That's just a personal preference

**Comment by Xorio71** [comment]
Yes, but at the same time, it means that the theme is not fully in Material You, which is a point that could be made.

**Comment by CarlosEsco** [comment]
> Yes, but at the same time, it means that the theme is not fully in Material You, which is a point that could be made.

How? Just because it's a surface color doesn't mean it's not material you.

**Comment by Xorio71** [comment]
I didn't say it's not Material You. I said it's missing elements of it.
